server/routes/api/messages.js

```diff
@@ -36,11 +36,27 @@ router.post("/", async (req, res, next) => {
       senderId,
       text,
       conversationId: conversation.id,
+      read,
     });
     res.json({ message, sender });
   } catch (error) {
     next(error);
   }
 });
 
+router.put("/:messageId", async (req, res, next) => {
+  const { messageId } = req.params;
+  const { read } = req.body
+  //console.log(`update request message id ${messageId}`);
+  try{
+    const message = await Message.findByPk(messageId)
+    message.read = read;
+    message.save();
+} catch (error) {
+  next(error);
+}
+  res.send('');
```

Comment:
Because this route does not return anything
```suggestion
  res.sendStatus(204);
```

Reply:
Makes sense. Done